toolbox/fdc3-compliance/src/test/fdc3.addContextListener.ts

```diff
@@ -0,0 +1,7 @@
+describe("fdc3.addContextListener", async () => {
+  it("Method is callable", async () => {
+    await window.fdc3.addContextListener("fdc3.contact", (info: any) => {
```

Comment:
Shouldn't we also add clean up logic (e.g. inside of an `afterEach()`) to prevent memory leaks?